react-components/src/components/Image360Details/Image360Details.tsx

```diff
@@ -0,0 +1,85 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { useState, type ReactElement, useCallback, useEffect } from 'react';
+import styled from 'styled-components';
+import { Image360HistoricalDetails } from '../Image360HistoricalDetails/Image360HistoricalDetails';
+import { useReveal } from '../..';
+import { type Image360 } from '@cognite/reveal';
+import { Button } from '@cognite/cogs.js';
+
+export function Image360Details(): ReactElement {
+  const viewer = useReveal();
+  const [enteredEntity, setEnteredEntity] = useState<Image360 | undefined>();
+  const [is360HistoricalPanelExpanded, setIs360HistoricalPanelExpanded] = useState<boolean>(false);
+  const handleExpand = useCallback((isExpanded: boolean) => {
+    setIs360HistoricalPanelExpanded(isExpanded);
+  }, []);
+
+  const clearEnteredImage360 = (): void => {
+    setEnteredEntity(undefined);
+  };
+
+  const exitImage360Image = (): void => {
+    viewer.exit360Image();
+  };
+
+  const collections = viewer.get360ImageCollections();
+
+  useEffect(() => {
+    collections.forEach((collection) => {
+      collection.on('image360Entered', setEnteredEntity);
+      collection.on('image360Exited', clearEnteredImage360);
+    });
+    return () => {
+      collections.forEach((collection) => {
+        collection.off('image360Entered', setEnteredEntity);
+        collection.off('image360Exited', clearEnteredImage360);
+      });
+    };
+  }, [viewer, collections]);
+
+  return (
+    <>
+      {enteredEntity !== undefined && (
+        <>
+          <Image360HistoricalPanel isExpanded={is360HistoricalPanelExpanded}>
+            <Image360HistoricalDetails
+              viewer={viewer}
+              image360Entity={enteredEntity}
+              onExpand={handleExpand}
+            />
+          </Image360HistoricalPanel>
+          <ExitButtonContainer>
+            <Button icon="CloseLarge" type="tertiary" onClick={exitImage360Image} />
+          </ExitButtonContainer>
+        </>
+      )}
+    </>
+  );
+}
+
+const ExitButtonContainer = styled.div`
+  position: absolute;
+  right: 20px;
+  top: 20px;
+  background-color: #ffffff;
+  height: 35px;
+  width: 35px;
+  border-radius: 6px;
```

Comment:
Details panel, toolbar and all other buttons in FDX has 8px radius, so it would be nice to have the same feel across all buttons. You will also need to create `StyledButton` with 8px border-radius.